starboard/nplb/player_test_util.cc

```diff
@@ -147,6 +147,44 @@ std::vector<SbPlayerTestConfig> GetSupportedSbPlayerTestConfigs(
   return test_configs;
 }
 
+std::vector<SbPlayerTestConfig> GetVerticalVideoTestConfigs() {
+  const char* kEmptyName = NULL;
+
+  // TODO: Add these files.
+  const char* kFilenames[] = {"1080_1920_avc.dmp", "1440_2560_av1.dmp",
+                              "3840_2160_av1.dmp"};
+
+  const SbPlayerOutputMode kOutputModes[] = {kSbPlayerOutputModeDecodeToTexture,
+                                             kSbPlayerOutputModePunchOut};
+
+  std::vector<const char*> video_files;
+  video_files.push_back(kEmptyName);
```

Comment:
I think we don't need to test empty video file here.

Reply:
Done